src/routes.ts

```diff
@@ -114,6 +114,6 @@ export default [
   {
     // default/fallback route
     path: '*',
-    redirect: '/'
+    redirect: '/dashboard'
```

Comment:
We refer to dashboard in a few spots but haven't defined the route as such. Therefore navigation to `dashboard` was always caught by the fallback route and subsequently root. 
Since this UI has several named sub-roots, it would be a good idea to be explicit about our dashboard.

Reply:
That was there so the root URL would be the dashboard (not "/dashboard" in the brower URL). Does this still work?

Reply:
It would display as `BC0870718/dashboard` by naming the route. 
I can remove this though, will look into this a little deeper, as i'm worried the fallback route is called vs our root route sometimes. Trying to make sense of it yet :)

Reply:
Removed this 👍